### DISCUSSION TITLE
SOURCE CODE PRIVATED UNTIL 1.0 RELEASES AND ALL ACTION BUILDS BEFORE 1.0 DELETED

### DISCUSSION THREAD

**Discussion by sonicfan888** [sonicfan888/92739374937292#6] [Announcements]
The source code has been privated due to it appearing on Bing.
Source code will be unprivated when 1.0 is released and all action builds before 1.0 have been deleted.

Have an good day.

**Comment by itsDARK768** [comment]
Ass

**Reply by sonicfan888** [reply]
lanky kong ate ass
